docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-designing-a-case-proc.adoc

```diff
@@ -1,2 +1,99 @@
-[id='']
-= TODO
\ No newline at end of file
+[id='case-management-designing-a-case-proc']
+= Designing a case
+
+Cases are designed using the *Case definition* design editor in {CENTRAL}. The case design is the basis of case management and sets out the specific goals and tasks for each case. The design can be added to and modified dynamically during runtime.
+
+To demonstrate how to design a case, {PRODUCT} includes the IT Orders sample project in {CENTRAL}, which includes the following case definition:
+
+image::itorders-orderhardware-process.png[IT Orders case design]
+
+You can refer to and interact with this case definition by importing the sample project in {CENTRAL}. 
+
+.Prerequisite 
+* You have followed the procedure in <<case-management-creating-a-case-proc>> to create a new case in {CENTRAL}.
+
+.Procedure
+. *Create a new case definition.*
```

Comment:
No bold. I know what you're going for, where you have a bolded super step with sub-steps to achieve it, but it's not consistent here, plus even when it is, I find (personally) that just using the actual steps is just fine. Example, I'd next what you have as step one and just make the "In the Assets window" line step 1.